lib/rpm_package.py

```diff
@@ -255,6 +259,35 @@ def _load(self):
         except TypeError:
             raise exception.PackageDescriptorError(package=self.name)
 
+    def get_spec_macros(self):
+        """
+        Get command line string to define spec file macros externally.
+
+        Returns:
+            str: string to be appended to the rpmbuild command
+        """
+        macros_string = ""
+        for package_attribute, macro_name in (
+                PACKAGE_METADATA_TO_RPM_MACRO_MAPPING.items()):
+            subnode = self.package_data
+            for subnode_id in package_attribute:
+                try:
+                    subnode = subnode[subnode_id]
+                except (KeyError, IndexError):
+                    LOG.debug(
```

Comment:
This logs a debug msg for each mappings which is not found. IMHO it should log an error msg (and possibly cancel the build instead of failing silently) if none of the mappings are found.